TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/210_Motion_Vectors/CaptureMotionVectorsPass.cs

```diff
@@ -35,7 +35,9 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
         using (new ProfilingScope(cmd, m_ProfilingSampler))
         {
             m_Material.SetFloat("_Intensity", m_intensity);
-            cmd.Blit(m_CameraColorTarget, m_CameraColorTarget, m_Material);
+            cmd.SetRenderTarget(m_CameraColorTarget, m_CameraColorTarget);
+            cmd.ClearRenderTarget(false, true, Color.clear);
```

Comment:
you should not use SetRenderTarget in a URP render pass. Instead you should call ConfigureTarget from withing ScriptableRenderPass.OnCameraSetup this will make sure the render pass merging works for this pass.

Reply:
If this is pass only for testing?

Reply:
This pass is only for testing